library/std/src/macros.rs

```diff
@@ -99,9 +99,9 @@ macro_rules! println {
     () => {
         $crate::print!("\n")
     };
-    ($($arg:tt)*) => {
-        $crate::io::_print($crate::format_args_nl!($($arg)*))
-    };
+    ($($arg:tt)*) => {{
```

Comment:
Given the back and forth here, it might be worth adding a comment that explains why a double brace is used?